src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -253,6 +255,14 @@ private static String createQuery(Map<String, Object> params)
 							nestedEntry.getValue());
 				}
 				flatParams.putAll(flattenParams(flatNestedMap));
+			} else if (value instanceof List<?>) {
+				Map<String, Object> flatNestedMap = new LinkedHashMap<String, Object>();
+				int index = 0;
+				for (Object element : (List<?>) value) {
```

Comment:
Changed to use an iterator.

How do you mean by the allocation of a new object? I thought `for (x : y)` in Java expanded into a loop using the iterator methods.